pkg/cmd/util/output/backup_describer.go

```diff
@@ -885,7 +888,7 @@ type volumesByPod struct {
 
 // Add adds a pod volume with the specified pod namespace, name
 // and volume to the appropriate group.
-func (v *volumesByPod) Add(namespace, name, volume, phase string, progress veleroapishared.DataMoveOperationProgress) {
+func (v *volumesByPod) Add(namespace, name, volume, phase string, progress veleroapishared.DataMoveOperationProgress, incremental bool) {
```

Comment:
I suggest we change this new parameter name `incremental`. We can just call it `forBackup`.
Reasons:
1. It is confusing, it is not true that backup is always incremental and restore is always full
2. Even for a full backup, `incrementalSize` vs. `fullSize` is still meaningful